blog/_posts/2019-09-05-put-down-particle-accelerator.md

```diff
@@ -0,0 +1,78 @@
+
+MDEs will be reactive to provide the best of both IDEs and Kubernetes. Like IDEs, MDEs will watch your filesystem to upate when you save. Like Kubernetes they'll watch your cluster to respond to unexpected events. The same way Kubernetes frees you from restarting a process, MDEs free you from remembering to do a build or refresh a log.
+
+## Early MDEs
+One popular declarative description of how to build and deploy your microservice app is Docker Compose. [Docker Compose](https://docs.docker.com/compose/) is a proto-MDE: it can build and start, but won't react as you edit with incremental updates. To be clear: we love Docker Compose! It's incredibly popular, a testament to the value it delivers, and users can hack in updates via mount points. Other proto-MDEs work outside containers, like foreman, the cleverly-named go-implementetation goreman, and modd offer similar functionality outside containers.
+
+I work building [Tilt](https://tilt.dev), a Kubernetes-focused MDE. If you use Kubernetes in production, you need Tilt in development. There are other MDE projects, like [Skaffold](https://skaffold.dev), [Tekton](https://tekton.dev) (sort of an MDE), [Garden](https://garden.io), [Okteto](https://okteto.com), or [Kelda](https://kelda.io).
+
+I'm especially proud of a few Tilt features that really exemplify what MDEs can be, how they can offer a better experience, and how Tilt's going to grow:
+* The [Tiltfile](https://docs.tilt.dev/tutorial.html) is an ergonomic way to define how to assemble the molecule of your app, from the simple (our [tutorial](https://docs.tilt.dev/tutorial.html) highlights getting started with just Dockerfiles and Kubernetes YAML) or complex (using [`local`](https://docs.tilt.dev/tiltfile_concepts.html) to call an existing yaml generator).
+* [`custom_build`](https://docs.tilt.dev/custom_build.html) lets you call out to an existing build tool. (A community member was able to integrate their Bazel builds without reuqiring any changes to Tilt)
+* Tilt's localhost Web UI is a Heads-Up Display that gives you context across your app so you see problems faster, without having to play 20 questions with Kubectl
+* [Live Update](https://blog.tilt.dev/2019/04/02/fast-kubernetes-development-with-live-update.html) lets you skip image builds altogether: it updates running containers with new code, even for compiled languages.
+
+## MDEs: the Right Tool for the Job
+An MDE is going to supercharge your development in the times when unit tests aren't enough to understand what's happening across your app. Tilt's still just a fraction of what it will be; if you want to help us realize this vision and have it for your inner loop, try Tilt and tell us what you think.
```

Comment:
> when unit tests aren’t enough to understand what’s happening across your app

seems like an unhelpful comparison / i'm not sure that's a thing i'd ever try to do. Maybe:
> ...when running 5 different `start.sh` scripts isn't enough to understand...